test/e2e.method.send.js

```diff
@@ -1,7 +1,7 @@
 var assert = require('assert');
-var Web3 = require('../packages/web3');
 var Basic = require('./sources/Basic');
 var utils = require('./helpers/test.utils');
+var Web3 = (global.window) ? require('../packages/web3/dist/web3.min') : require('../packages/web3');
```

Comment:
A small possible improvement here would be to have a ``getWeb3`` helper function instead of duplicating this condition anywhere. :)

Reply:
👍 Added in c116e879b1f7b0c836a2812f3b9c207410fe52ea